### ISSUE TITLE
Could not locate that index-pattern-field (id: src_ip.keyword)

### ISSUE THREAD

**Issue by MyCodeRocks** [evaluationcopy/pfsense-suricata-elk-docker#2]
Hi there 
followed the steps to the tee but in the dashboard I keep getting a few errors (below) can you pls help or point in the right direction:
Could not locate that index-pattern-field (id: src_ip.keyword)
Could not locate that index-pattern-field (id: geoip.country_code3.keyword)

Thats for both dashboards

**Comment by krolingo** [comment]
Is there a known working solution? I have the same problem :/

**Comment by MyCodeRocks** [comment]
@krolingo no not yet I am going to spend some time looking at this. The GeoIP mapping in the logstash index in Elasticsearch doesn't add these fields.

A week ago when I had time I was looking at creating a new logstash index template to accommodate but ran out of time and think it is the wrong approach.

Looking at using Filebeat from Elsastic.co to pump the info into. Elastic then have to create some dashboards. The dashboards are not the issue it's the Grok and output file that's the issue right now.

**Comment by krolingo** [comment]
yes the issue is with the pfsense2-4.grok definitions. if you find what's causing the definitions to fail please post your solution.

**Comment by AlexNikolovLON** [comment]
The issue come from the export and then import of the dashboard 

Right now, when we export a dashboard, the index pattern(s) does not get exported along with it. So the import process requires the user to create the correct index patterns(s) manually and then re-import
